src/generate/project.rs

```diff
@@ -176,21 +176,22 @@ history.txt
 name = "{}"
 authors = []
 telemetry = {}
+cache_dir = "./.requirements"
```

Comment:
If this goes into the current directory by default, lets add it to the .gitignore also.

Reply:
I'm ambivalent about defaulting to current dir, but it will be more convenient for developers when dealing with requirements.

Reply:
Your feedback was addressed in https://github.com/hirosystems/clarinet/pull/333/commits/c03549d1fa3804faa9fbd19f01e0933d0051ae0f